### ISSUE TITLE
Error: "Refused to display [...] in a frame because it set 'X-Frame-Options' to 'deny'"

### ISSUE THREAD

**Issue by fedorov** [OHIF/Viewers#1711]
### Describe the Bug

Trying to load a study that has a SEG referencing an image, but without corresponding image in that same study, there is a cryptic runtime error:

![image](https://user-images.githubusercontent.com/313942/81115714-27d93e80-8ef2-11ea-84c3-b4bcc5e3d0af.png)

### What steps can we follow to reproduce the bug?

Populate a Google Healthcare DICOM store with a single SEG series that references another series in `ReferencedSeriesSequence`, and try to open the study with that SEG in OHIF Viewer.

Observed with 3.11.7.

**Comment by JamesAPetts** [comment]
Do you have a link to a study on the sandbox causing these issues?

**Comment by fedorov** [comment]
I believe it is this one: https://idc-sandbox-000.firebaseapp.com/projects/idc-tcia/locations/us/datasets/issues/dicomStores/issue-2-kits/study/1.3.6.1.4.1.14519.5.2.1.3344.2526.267929376932212971281995255548

but it is now failing with a different error

![image](https://user-images.githubusercontent.com/313942/82061167-58f60380-9696-11ea-8342-863727bbc465.png)

**Comment by sedghi** [comment]
@igoroctaviano What is the status of this?

**Comment by igoroctaviano** [comment]
@sedghi looks like it's still a problem in v2 and also for v3: 
<img width="1622" alt="Screenshot 2023-10-11 at 15 17 37" src="https://github.com/OHIF/Viewers/assets/13886933/39136dc0-6b5c-4a61-bdc3-85a23524fbbc">

**Comment by igoroctaviano** [comment]
> @sedghi looks like it's still a problem in v2 and also for v3: <img alt="Screenshot 2023-10-11 at 15 17 37" width="1622" src="https://user-images.githubusercontent.com/13886933/274377810-39136dc0-6b5c-4a61-bdc3-85a23524fbbc.png">

@fedorov do we want to prioritize this issue? (this was created for v2 for looks like it applies to v3)

**Comment by sedghi** [comment]
Actually this`ReferencedSeriesSequence` is causing problem in other data in TCIA with SEG that recently got added by SPL. As far as I understand this is a required tag. 

Data: https://www.dropbox.com/s/2sbi42khfesmb86/ReMIND-001.zip?dl=0

@fedorov This is the ReMIND data from Tina. Do you happen to know if that `ReferencedSeriesSequence ` is necessary to be available for SEG? seems like so here https://dicom.innolitics.com/ciods/segmentation/common-instance-reference/00081115

**Comment by fedorov** [comment]
>  Do you happen to know if that ReferencedSeriesSequence  is necessary to be available for SEG?

No, it is not required when segmentation geometry (ImageOrientation/Position/Spacing etc) is described in the SEG instance, which is typically the case.

**Comment by fedorov** [comment]
We just tested this in OHIF demo viewer local mode with the dataset from dropbox above, and the error was different

<img width="1523" alt="image" src="https://github.com/OHIF/Viewers/assets/313942/ed86d004-4d65-45b8-9a55-c7b9ec967dd8">

**Comment by sedghi** [comment]
Yeah I fixed the error that prevented the viewer from showing up, now only the SEG is not loadable. But still not sure what should we do regarding that tag missing (which is required as you mentioned)

**Comment by igoroctaviano** [comment]
> Yeah I fixed the error that prevented the viewer from showing up, now only the SEG is not loadable. But still not sure what should we do regarding that tag missing (which is required as you mentioned)

The case here is that the tag is there but references a series not part of the current study. In this case, it should fallback to a frame of reference (it still should be able to render fine by a frame of reference even though the tag references a missing series)
